Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -57,9 +57,15 @@ function Initialize-AzureSubscription {
             $additional['CurrentStorageAccountName'] = $StorageAccount
         }
 
+        $EnvironmentName = "AzureCloud"
```

Comment:
Can you add a unit test? I put a lot of effort to add L0 coverage for this module.